data/store/mongo/mongo.go

```diff
@@ -62,3 +68,8 @@ func (s *Store) NewAlertsRepository() alerts.Repository {
 	r := alertsRepo(*s.Store.GetRepository("alerts"))
 	return &r
 }
+
+func (s *Store) NewDeviceTokensRepository() devicetokens.Repository {
+	r := deviceTokensRepo(*s.Store.GetRepository("deviceTokens"))
```

Comment:
[nit] I don't think the local variable is necessary here.

Reply:
I'd remove it if I could, but the Go compiler disagrees with you. :P
> invalid operation: cannot take address of deviceTokensRepo(*s.Store.GetRepository("deviceTokens")) (value of type deviceTokensRepo) (compile)

Reply:
Sorry, to be more clear, I was suggesting using `{}` brackets for construction.
```
    return &deviceTokensRepo{*s.Store.GetRepository("deviceTokens")}
```
but feel free to leave it as-is.